torch/csrc/autograd/functions/batch_normalization.cpp

```diff
@@ -111,7 +117,7 @@ auto BatchNormBackward::apply(const variable_list& grad_outputs) -> variable_lis
   }
 
   std::unique_ptr<Tensor> grad_bias;
-  if (bias) {
+  if (needs_input_grad(2) || use_cudnn) {
     grad_bias = bias->newTensor();
     grad_bias->resizeAs(*bias);
     grad_bias->zero();
```

Comment:
It seems that there's another `zero()` call 2 lines below. Can you fix that please?